### PR TITLE
Correct spelling mistakes.

### PR DESCRIPTION
**Pull request by EdwardBetts**

### CODE REVIEW

File: src/wallet/api/wallet2_api.h [hunk 1]
```diff
@@ -720,7 +720,7 @@ struct Wallet
      * \param signature - the signature
      * \return true if the signature verified, false otherwise
      */
-    virtual bool verifySignedMessage(const std::string &message, const std::string &addres, const std::string &signature) const = 0;
+    virtual bool verifySignedMessage(const std::string &message, const std::string &address, const std::string &signature) const = 0;
```
Comment (dEBRUYNE-1) [inline src/wallet/api/wallet2_api.h]: This might break stuff further down the line.

File: tests/core_tests/chain_switch_1.cpp [hunk 2]
```diff
@@ -151,9 +151,9 @@ bool gen_chain_switch_1::check_split_not_switched(cryptonote::core& c, size_t ev
   CHECK_EQ(1, tx_pool.size());
 
   std::vector<size_t> tx_outs;
-  uint64_t transfered;
-  lookup_acc_outs(m_recipient_account_4.get_keys(), tx_pool.front(), get_tx_pub_key_from_extra(tx_pool.front()), get_additional_tx_pub_keys_from_extra(tx_pool.front()), tx_outs, transfered);
-  CHECK_EQ(MK_COINS(13), transfered);
+  uint64_t transferred;
```
Comment (dEBRUYNE-1) [inline tests/core_tests/chain_switch_1.cpp]: This might break stuff further down the line.

File: tests/core_tests/chain_switch_1.cpp [hunk 3]
```diff
@@ -202,9 +202,9 @@ bool gen_chain_switch_1::check_split_switched(cryptonote::core& c, size_t ev_ind
   CHECK_TEST_CONDITION(!(tx_pool.front() == m_tx_pool.front()));
 
   std::vector<size_t> tx_outs;
-  uint64_t transfered;
-  lookup_acc_outs(m_recipient_account_2.get_keys(), tx_pool.front(), tx_outs, transfered);
-  CHECK_EQ(MK_COINS(7), transfered);
+  uint64_t transferred;
```
Comment (dEBRUYNE-1) [inline tests/core_tests/chain_switch_1.cpp]: This might break stuff further down the line.

File: tests/core_tests/ring_signature_1.cpp [hunk 4]
```diff
@@ -336,10 +336,10 @@ bool gen_ring_signature_big::check_balances_2(cryptonote::core& c, size_t ev_ind
   }
 
   std::vector<size_t> tx_outs;
-  uint64_t transfered;
+  uint64_t transferred;
   const transaction& tx = boost::get<transaction>(events[events.size() - 3]);
-  lookup_acc_outs(m_alice_account.get_keys(), boost::get<transaction>(events[events.size() - 3]), get_tx_pub_key_from_extra(tx), get_additional_tx_pub_keys_from_extra(tx), tx_outs, transfered);
-  CHECK_EQ(m_tx_amount, transfered);
+  lookup_acc_outs(m_alice_account.get_keys(), boost::get<transaction>(events[events.size() - 3]), get_tx_pub_key_from_extra(tx), get_additional_tx_pub_keys_from_extra(tx), tx_outs, transferred);
+  CHECK_EQ(m_tx_amount, transferred);
```
Comment (dEBRUYNE-1) [inline tests/core_tests/ring_signature_1.cpp]: This might break stuff further down the line.

File: tests/core_tests/chain_switch_1.cpp [hunk 5]
```diff
@@ -151,9 +151,9 @@ bool gen_chain_switch_1::check_split_not_switched(cryptonote::core& c, size_t ev
   CHECK_EQ(1, tx_pool.size());
 
   std::vector<size_t> tx_outs;
-  uint64_t transfered;
-  lookup_acc_outs(m_recipient_account_4.get_keys(), tx_pool.front(), get_tx_pub_key_from_extra(tx_pool.front()), get_additional_tx_pub_keys_from_extra(tx_pool.front()), tx_outs, transfered);
-  CHECK_EQ(MK_COINS(13), transfered);
+  uint64_t transferred;
+  lookup_acc_outs(m_recipient_account_4.get_keys(), tx_pool.front(), get_tx_pub_key_from_extra(tx_pool.front()), get_additional_tx_pub_keys_from_extra(tx_pool.front()), tx_outs, transferred);
+  CHECK_EQ(MK_COINS(13), transferred);
```
Comment (dEBRUYNE-1) [inline tests/core_tests/chain_switch_1.cpp]: This might break stuff further down the line.

File: src/p2p/net_node.h [hunk 6]
```diff
@@ -119,7 +119,7 @@ namespace nodetool
     size_t get_outgoing_connections_count();
     peerlist_manager& get_peerlist_manager(){return m_peerlist;}
     void delete_connections(size_t count);
-    virtual bool block_host(const epee::net_utils::network_address &adress, time_t seconds = P2P_IP_BLOCKTIME);
+    virtual bool block_host(const epee::net_utils::network_address &address, time_t seconds = P2P_IP_BLOCKTIME);
```
Comment (dEBRUYNE-1) [inline src/p2p/net_node.h]: This might break stuff further down the line.

**Comment by dEBRUYNE-1** [comment]
In general, I think most contributors are not in favor of correcting spelling mistakes except if they occur in comments.

**Comment by jtgrassie** [comment]
I'd agree with @dEBRUYNE-1 Personally I don't see the point in even creating PR's for spelling mistakes in comments. The spelling mistakes "fixed" in this PR are hardly show-stoppers (e.g. they not ambiguous to the point of causing confusion), they are very minor.

File: src/p2p/net_node.h (hunk 6 above)
Comment (stoffu) [reply src/p2p/net_node.h]: It makes no difference since this is only a function declaration. The argument name is never referenced.

**Comment by moneromooo-monero** [comment]
I'd been wondering whether to ask for the code changes to be taken out. Guess I am since others seem to be in agreement. I'm fine with the comments fixed.

**Comment by leonklingele** [comment]
What's the problem with fixing var names inside functions and argument names inside function declarations?

**Comment by moneromooo-monero** [comment]
They add unnecessary code changes. It's a bit subjective, but we don't want large non fonctional code changes, so there has to be a line somewhere, and so that's where it is here, mostly because it's easier than to discuss where it is at each commit adding some of that. There's a lot of mispelt stuff in the code fwiw.

**Comment by leonklingele** [comment]
OT: I was about to create a PR which removes all trailing whitespaces (there are a ton!), guess that won't get through?

**Comment by hyc** [comment]
@leonklingele please reread CONTRIBUTING.md. This is all already explicitly addressed. And No.

**Comment by leonklingele** [comment]
@hyc doesn't seem to be the case. 

> such as random whitespace changes or reindentation

It's not a random change, the PR will be dedicated to remove trailing spaces.

**Comment by hyc** [comment]
Patches should make functional improvements - fixing bugs or adding new features. Patches that consist entirely of cosmetic changes get rejected.

**Comment by leonklingele** [comment]
@hyc Removing trailing spaces __is__ useful and improves working with the code. Many editors delete trailing spaces by default (which is a good thing). This makes it harder to create patches for Monero with at least my environment and it sucks to interactively pick changes with Git.

**Comment by moneromooo-monero** [comment]
> >   such as random whitespace changes or reindentation

> It's not a random change, the PR will be dedicated to removing trailing spaces.

Oh, come on. Don't take the piss.

**Comment by moneromooo-monero** [comment]
Do you want to take out the code bits, so this can be merged ?

**Comment by iamsmooth** [comment]
@dEBRUYNE-1 "I think most contributors are not in favor of correcting spelling mistakes except if they occur in comments" 

I'm not even in favor of fixing spelling errors  in comments if that is the only thing being done. If the associated code is being substantively changed in some way then sure go ahead and fix the nearby comments at the same time. Otherwise this is just pollution of the change history, which is a critical issue when there are security considerations. More pollution means more opportunity for accidental _or malicious_ changes to be slipped in and lost in the noise.

**Comment by lh1008** [comment]
@iamsmooth I don´t agree. If some contributers won´t or don´t like changing spelling mistakes, others contributers should take care of spelling mistakes. I understand "More pollution means more opportunity for accidental or malicious changes to be slipped in and lost in the noise."...but not changing spelling mistakes considers coders don´t care if they misspell.

**Comment by stoffu** [comment]
@lh1008 You're missing the point. Monero is a security software. We care the most about whether the code is functional and every change is well intended. We care the least about spelling correctness.

**Comment by lh1008** [comment]
@stoffu ...yeah you are probably right. I've been working in some translations..., a misspell is an issue. I should not be giving any comments over here....sorry... :)

**Comment by iamsmooth** [comment]
@lh1008 if you are referring to translation of documents, web sites, etc. then I absolutely agree to get the spelling right. Even in UI which comes from code, again I agree it is important. But when we are talking about the _code itself_, including embedded comments, then I think the ease of maintenance (and security) of the code is paramount, and those considerations should take precedence over correcting spelling errors (and also other unnecessary formatting changes such as spacing).

**Comment by lh1008** [comment]
@iamsmooth you're right. Maybe this issue is only to take it as a consideration, nothing else... :)

**Comment by EdwardBetts** [comment]
I updated my patch to remove code changes.

**Comment by fluffypony** [comment]
@EdwardBetts please can you rebase this, there's a conflict. Thanks!

**Comment by EdwardBetts** [comment]
@fluffypony: Done.

**Pull request merged**